### ISSUE TITLE
[Bug]: CustomMatch is not settable on BarLink

### ISSUE THREAD

**Issue by panmona** [Megabit/Blazorise#5567]
### Blazorise Version

1.3

### What Blazorise provider are you running on?

None

### Link to minimal reproduction or a simple code snippet

```
<BarItem>
      <BarLink To="MyUrl"
               Match="Match.Custom"
               CustomMatch="(url) => true"
               Padding="Padding.Is2.OnX">
          <BarIcon Class="notranslate"
                   IconName="Blazorise.Icons.Material.MaterialIcons.People" />
          MyUrl
      </BarLink>
  </BarItem>
```

### Steps to reproduce

Compile the code :)

### What is expected?

As the BarLink allows to use Match.Custom, it should also expose the CustomMatch Func, just like the Link and Anchor components.

### What is actually happening?

Unfortunately this method is not exposed.

### Any additional comments?

Additionally I noticed that the type shown for "Match" in the "BarLink" documentation is not correct. It is of type Match and not of type bool: https://blazorise.com/docs/components/bar

For now we are using Anchor with the class "b-bar-link" as a workaround.

**Comment by stsrki** [comment]
It should be an easy fix, but we're going to do it in 1.6.

**Comment by stsrki** [comment]
#5567
